src/rf/apps/core/models.py

```diff
@@ -230,10 +243,6 @@ class LayerImage(Model):
         max_length=255,
         help_text='Filename of original file',
     )
-    s3_uuid = UUIDField(
```

Comment:
I would favor keeping the actual URI in-tact since we can use it to lookup images without having to perform string slicing (which I find kind of flimsy). Indexes should make up the performance difference.